custom_components/husqvarna_automower/translations/et.json

```diff
@@ -75,19 +184,51 @@
             "mower": {
                 "state": {
                     "cleaning": "Niidab"
+                },
+                "state_attributes": {
+                    "status": {
+                        "name": "Olek",
+                        "state": {
+                            "cleaning": "Niidab",
+                            "off": "Väljas",
+                            "unknown": "Teadmata",
+                            "stopped": "Peatunud",
+                            "not_applicable": "Pole kohalduv",
+                            "paused": "Pausil",
+                            "going_to_charging_station": "Läheb laadimisjaama",
+                            "leaving_charging_station": "Lahkub laadimisjaamast",
+                            "parked": "Pargitud",
+                            "park_override": "Parkimise tühistamine",
+                            "daily_limit": "Päevane limiit",
+                            "fota": "Püsivara uuendamine",
+                            "updating": "Uuendab",
+                            "powering_up": "Käivitub",
+                            "weather_timer": "Ilmataimer",
+                            "parked_until_further_notice": "Pargitud kuni edasise korralduseni"
+                        }
+                    },
+                    "action": {
+                        "name": "Tegevus",
+                        "state": {
+                            "force_mow": "Saada niitma",
+                            "not_active": "Pole aktiivne",
+                            "unknown": "Teadmata"
+                        }
+                    }
                 }
             }
         },
         "camera": {
-            "mower": {
+            "mower_cam": {
+                "name": "Kaart",
                 "state": {
                     "cleaning": "Niidab",
                     "docked": "Dokitud",
                     "error": "Viga",
-                    "idle": "[%key:common::state::idle%]",
-                    "off": "[%key:common::state::off%]",
-                    "on": "[%key:common::state::on%]",
-                    "paused": "[%key:common::state::paused%]",
+                    "idle": "Jõude",
```

Comment:
Aren't the common strings working?

Reply:
They are. I decided to follow the style in `en.json`.

Reply:
I think there was something wrong in this commit. https://github.com/Thomas55555/husqvarna_automower/commit/eca2a390a015b37f8f1ddb14e55862da4db7b9d2
Can you switch back to the common keys?